### PR TITLE
[MRG] feat: Add itemcount in project job overview & item job detail

### PR DESCRIPTION
**Pull request by joaquingx**
# Description

Add item count in job overview and item job detail

# Issue

* _Github Issue ID_.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] My code follows the style guidelines of this project.
- [ ] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [ ] New and existing tests pass locally with my changes.
- [ ] If this change is a core feature, I have added thorough tests.
- [ ] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/).
- [ ] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by joaquingx** [comment]
Job Detail page adds item count:
Screenshot 2024-10-27 at 14 04 18

Running jobs in project-job overview now displays the item count:
Screenshot 2024-10-27 at 14 13 54

File: estela-web/src/pages/JobDetailPage/index.tsx [hunk 1]
```diff
@@ -660,6 +660,7 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
             spiderName,
             totalResponseBytes,
             items,
+            itemsCount,
```
Comment (mgonnav) [inline estela-web/src/pages/JobDetailPage/index.tsx:663]: @joaquingx I think there's an issue here. We should obtain the `itemCount` in a similar manner as here: https://github.com/bitmakerla/estela/commit/3d39508ab0b30a30213c7f70dc0c40ff81e83eaf#diff-1763b2c090221df6b4323f653594f134a4a6ce3fd84f21e06140f5da2a66f08bR179

I have Redis stats set to update every 5 seconds, but the frontend shows a value on every page refresh, which doesn't make sense. The processing time doesn't update on every refresh, for example.

Comment (joaquingx) [reply estela-web/src/pages/JobDetailPage/index.tsx]: I was getting item count in DB, I changed to get it from the redis, please review it.

**Review by mgonnav — APPROVED** [review]
LGTM!

**Pull request merged**